tests/bootstrap.php

```diff
@@ -82,7 +82,7 @@
     ]
 ]);
 
-Cache::config([
+Cache::setConfig([
```

Comment:
Don't update tests/bootstrap.php. I use it in my plugins for running test suite and this change will break tests for < 3.4